src/contexts/Prices/PricesProvider.tsx

```diff
@@ -113,8 +113,11 @@ const PricesProvider: React.FC = ({ children }) => {
         const calculatedApy = dailyYield.multipliedBy(365)
 
         setFarmTwoApy(calculatedApy.toFixed(2))
-      }
-    )
+      })
+      .catch((error) => {
+        console.log(error)
+        setFarmTwoApy('0.00')
```

Comment:
@controtie How's this? Not much else we can do here except fail gracefully

Reply:
Yeah I think that should be good for now